src/pages/home/report/ReportAttachments.tsx

```diff
@@ -16,8 +16,7 @@ function ReportAttachments({route}: ReportAttachmentsProps) {
     const report = ReportUtils.getReport(reportID);
 
     // In native the imported images sources are of type number. Ref: https://reactnative.dev/docs/image#imagesource
-    const decodedSource = decodeURI(route.params.source);
-    const source = Number(decodedSource) || decodedSource;
+    const source = Number(route.params.source) || route.params.source;
```

Comment:
Is this change safe because you changed `encodeURI` to `encodeURIComponent` in the `REPORT_ATTACHMENT` route?

Reply:
Yes, it safe. I covered it here: https://github.com/Expensify/App/pull/37566/files#r1521953591